source/acquaman/AMGenericStepScanConfiguration.cpp

```diff
@@ -0,0 +1,187 @@
+#include "AMGenericStepScanConfiguration.h"
+
+#include "ui/acquaman/AMGenericStepScanConfigurationView.h"
+#include "acquaman/AMGenericStepScanController.h"
+
+#include <math.h>
+
+AMGenericStepScanConfiguration::AMGenericStepScanConfiguration(QObject *parent)
+	: AMStepScanConfiguration(parent)
+{
+	setName("Generic Scan");
+	setUserScanName("Generic Scan");
+}
+
+AMGenericStepScanConfiguration::AMGenericStepScanConfiguration(const AMGenericStepScanConfiguration &original)
+	: AMStepScanConfiguration(original)
+{
+	setName(original.name());
+	setUserScanName(original.name());
+
+	computeTotalTime();
+
+	for (int i = 0, size = scanAxes_.count(); i < size; i++){
```

Comment:
I would like it to be `for (int i = 0; i < scanAxes_.count(); i++)`. It will be the same in code efficiency.